src/pybind/mgr/dashboard_v2/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -54,7 +57,25 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges {
   // the details page, return false.
   @Input() beforeShowDetails: Function;
 
-  // Should be the function that will update the input data.
+  /**
+   * Auto reload time in ms - per default every 5s
+   * You can set it to 0, undefined or false to disable the auto reload feature in order to
+   * trigger 'fetchData' only once.
+   */
+  @Input() autoReload: any = 5000;
```

Comment:
I think this should not have a default value or `0`.

Reply:
5000 is used overall expect for one page with 30000. 5000 was also a request of @ricardoasmarques . If I would set it to 0 the default would be to not auto reload, but it's intended to be the opposite. If you don't want that on your table you can easily change it.